src/hotspot/share/prims/whitebox.cpp

```diff
@@ -2306,6 +2306,7 @@ WB_ENTRY(void, WB_VerifyFrames(JNIEnv* env, jobject wb, jboolean log, jboolean u
     tty_token = ttyLocker::hold_tty();
     tty->print_cr("[WhiteBox::VerifyFrames] Walking Frames");
   }
+  ResourceMark rm; // for verify
```

Comment:
Ran into an assert when testing because of a missing resource mark here, so I've added that as well.